docs/tutorials/trajectories.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+Handle Expert Trajectories
+==========================
+
+For imitation learning we need trajectories.
```

Comment:
I guess it depends on how much you want to explain this / who the audience is, but briefly explaining what a trajectory is could be a good start.

Reply:
Good point. I added a short explanation.